server/schemas.py

```diff
@@ -0,0 +1,88 @@
+from clear_my_record_backend.server import ma
+from clear_my_record_backend.server.models import *
+
+
+class ChargeSchema(ma.ModelSchema):
+    conviction = ma.Nested('ConvictionSchema', exclude=('charges', ))
+
+    class Meta:
+        model = Charge
+        fields = (
+            'id',
+            'conviction_id',
+            'charge',
+            'citation',
+            'sentence',
+            'class_type',
+            'charge_type',
+            'eligible',
+            'conviction'
+        )
+
+class ConvictionSchema(ma.ModelSchema):
+    charges = ma.Nested(ChargeSchema, many=True)
+    client = ma.Nested('ClientSchema', exclude=('convictions', ))
+
+    class Meta:
+        model = Conviction
+        fields = (
+            'id',
+            'client_id',
+            'client',
+            'case_number',
+            'agency',
+            'court_name',
+            'court_city_county',
+            'judge',
+            'record_name',
+            'release_status',
+            'release_date',
+            'charges',
+        )
+
+class ClientSchema(ma.Schema):
+    convictions = ma.Nested(ConvictionSchema, many=True, exclude=('client', ))
+    # exclude clients to avoid recursion issues lmao
+    user = ma.Nested('UserSchema', exclude=('clients', ))
+
```

Comment:
Need to figure out way to get `enum.KEY.name` to be output for schema for client_type, user_type, etc. Or change that column type.

Reply:
We should keep things as flexible as possible.  I do not know if the user really knows what they want at this point.